### ISSUE TITLE
decode=utf-8 and INFO: 'str' object has no attribute 'decode'

### ISSUE THREAD

**Issue by dpbriggs** [laixintao/iredis#260]
Hey! There's a type error if you use `--decode=utf-8` alongside `INFO`.

```bash
➜  redis-oxide git:(master) ✗ iredis info            
# Server
redis_version:5.0.7
... truncated ...

➜  redis-oxide git:(master) ✗ iredis --decode=utf-8 info
(error) 'str' object has no attribute 'decode'
```

I couldn't quite figure out how to run the project locally but it appears this is the only area in the code base that runs INFO: https://github.com/laixintao/iredis/blob/master/iredis/client.py#L80

**Comment by laixintao** [comment]
Hi @dpbriggs , thanks for reporting, will fix

**Comment by laixintao** [comment]
Hi @dpbriggs 

I released verion v1.0.1, please upgrade your iredis and see if it still happens : )

https://pypi.org/project/iredis/1.0.1/
